src/sagemaker/jumpstart/factory/estimator.py

```diff
@@ -189,6 +193,7 @@ def get_init_kwargs(
     estimator_init_kwargs = _add_metric_definitions_to_kwargs(estimator_init_kwargs)
     estimator_init_kwargs = _add_estimator_extra_kwargs(estimator_init_kwargs)
     estimator_init_kwargs = _add_role_to_kwargs(estimator_init_kwargs)
+    estimator_init_kwargs = _add_env_to_kwargs(estimator_init_kwargs)
```

Comment:
what happens if the customer passes an [environment](https://github.com/aws/sagemaker-python-sdk/blob/fdc0ac106524fc615a2cd333ffdb1065f6ce4785/src/sagemaker/jumpstart/estimator.py#L96) argument?

Reply:
These environment variables will be added to the environment variables retrieved from metadata